src/app/routes/_message.router.js

```diff
@@ -4,6 +4,21 @@ import moment from 'moment';
 
 export default (app, router, io, admin) => {
 
+  function meditatedRecently(user) {
```

Comment:
You could create another file which has the following contents:
```js
export default (user) => {
// code inside meditatedRecently
}
```
This file can then be used like this:
```js
import meditatedRecently form './thefile.js';
meditatedRecently(passingUser);
```